orca-api/src/main/java/com/netflix/spinnaker/orca/api/preconfigured/jobs/PreconfiguredStageParameter.java

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2017 Schibsted ASA.
+ * Copyright 2020 Netflix, Inc.
```

Comment:
Mind these copyright changes. Year updates are fine, but we should not be changing the organization.

Reply:
Thought I got everything . Missed it :(